### DISCUSSION TITLE
Temporizzare l'accensione di una luce

### DISCUSSION THREAD

**Discussion by osopensare** [indomus/forum#2627] [HUB: Home Assistant]
Buongiorno a tutti! Da poco ho installato HAsto cercando senza riuscire a temporizzare una luce (da fare poi su tutte). La luce è stata automatizzata con il sistema Bticino e vedo sulla plancia tutte le luci. In pratica vorrei semplicemente che se accendo la luce, dopo un 1 minuto si spenga automaticamente. Per fare questo ho creato un'automazione.
In "Attivazioni" ho messo il dispositivo che mi interessa in stato attivato, in "Condizioni" Entita il nome della luce Attribuito stato on per 1 min, in "Azioni" Disattiva luce

![image](https://github.com/indomus/forum/assets/144629952/6cc4b643-d882-4eca-8260-d2af685018a5)

![image](https://github.com/indomus/forum/assets/144629952/6aad2cfd-87dd-4134-9d05-57a726c799f7)

Tutto questo non funziona.

Mi potete aiutare?

**Comment by KC-inDomus** [comment]
devi mettere un'azione delay per l'attesa dello spegnimento, e togliere quella condizione, ché non c'entra nulla.

**Reply by osopensare** [reply]
Grazie.
Se ho capito bene dovrebbe essere cosi:
![immagine](https://github.com/indomus/forum/assets/144629952/daf82081-8459-46da-88ee-9164416004c5)
 
Ma a volte funziona e a volte non.
E comunque non certo dopo 15 s ma alcuni minuti dopo.

Dove sto sbagliando?

**Reply by osopensare** [reply]
Nessuno sa aiutarmi?
nessuno a questa esigenza?
o meglio a tutti funziona questa automazione?

**Reply by blakdog1978** [reply]
Stai sbagliando tutto. 
![Screenshot_2023-10-07-17-02-02-057_io homeassistant companion android](https://github.com/indomus/forum/assets/36457031/e913b35a-9dba-4c20-85e2-b2f985c11731)
Ti basta una cosa così
